CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [v3.0.0](https://github.com/voxpupuli/puppet-varnish/tree/v3.0.0) (2023-02-27)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-varnish/compare/d55e143663f24b4f5efd8a9628a3d0173264609b...v3.0.0)
+
+**Merged pull requests:**
+
+- Added support for new OS   - Dropped support outdated OS   - Moved VCL Subclasses \(acl, acl\_member, backend, director, probe, selector\)   - Added support for varnish 6   - Added support for varnish-plus / Varnish Enterprise [\#4](https://github.com/voxpupuli/puppet-varnish/pull/4) ([voxel01](https://github.com/voxel01))
```

Comment:
to make this a bit more beautiful, I created https://github.com/voxpupuli/puppet-varnish/issues/7. The next from that issue should be added to the changelog automatically. Please regenerate it.

Reply:
if this doesn't work properly, we maybe need to rename the milestone from v3.0.0 to 3.0.0.